config/dba.neon

```diff
@@ -1,26 +1,35 @@
 includes:
-	- stubFiles.neon
-	- extensions.neon
+    - stubFiles.neon
+    - extensions.neon
 
 services:
-	-
-		class: staabm\PHPStanDba\Rules\PdoStatementExecuteMethodRule
-		tags: [phpstan.rules.rule]
+    -
+        class: staabm\PHPStanDba\Rules\SyntaxErrorInPreparedStatementMethodRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            classMethods:
+                - 'Doctrine\DBAL\Connection::executeQuery'
+                - 'Doctrine\DBAL\Connection::executeCacheQuery'
+                - 'Doctrine\DBAL\Connection::executeStatement'
 
-	-
-		class: staabm\PHPStanDba\Rules\SyntaxErrorInQueryMethodRule
-		tags: [phpstan.rules.rule]
-		arguments:
-			classMethods:
-				- 'PDO::query#0'
-				- 'PDO::prepare#0'
-				- 'mysqli::query#0'
-				- 'Doctrine\DBAL\Connection::query#0'
+    -
+        class: staabm\PHPStanDba\Rules\PdoStatementExecuteMethodRule
+        tags: [phpstan.rules.rule]
 
-	-
-		class: staabm\PHPStanDba\Rules\SyntaxErrorInQueryFunctionRule
-		tags: [phpstan.rules.rule]
-		arguments:
-			functionNames:
-				- 'Deployer\runMysqlQuery#0'
-				- 'mysqli_query#1'
+    -
+        class: staabm\PHPStanDba\Rules\SyntaxErrorInQueryMethodRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            classMethods:
+                - 'PDO::query#0'
+                - 'PDO::prepare#0'
+                - 'mysqli::query#0'
+                - 'Doctrine\DBAL\Connection::query#0'
```

Comment:
IMO you could remove `query` and only support the latest APIs. Or you keep it but then I'd also add `exec` here and `executeUpdate` above in `SyntaxErrorInPreparedStatementMethodRule` for completeness.

Reply:
correct.. its just a few lines of config.. lets support them: https://github.com/staabm/phpstan-dba/pull/117